fiftyone/server/routes/temporal_tags.py

```
@@ -0,0 +1,462 @@
+    return value
+
+
+def _first_present(data: dict, *fields):
+    for field in fields:
+        if field in data:
+            return data[field]
+
+    return None
+
+
+def _query_values(params, *fields):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_query_values`

(ANN202)